docs/harvesting.md

```diff
@@ -251,6 +257,22 @@ This will create a new package with a harvester skeleton on which you can start
 You may take a look at the [existing backends][backends-repository] to see exiting implementations.
 
 
+## Debugging
```

Comment:
I think you may want to document this dedicated command instead?
```
udata harvest run <source_id>
```

Reply:
Incredible, this was there all along, I didn't realize 😬